README.md

```diff
@@ -164,6 +164,7 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `workers.targetCPUUtilization`                        | Target CPU Utilization of workers                                                                            | `80`                                              |
 | `workers.targetMemoryUtilization`                     | Target Memory Utilization of workers                                                                         | `80`                                              |
 | `workers.safeToEvict`                                 | Allow Kubernetes to evict worker pods if needed (node downscaling)                                           | `true`                                            |
+| `workers.extraContainers`                             | Add additional containers to worker pod(s)                                                                   | `[]`                                              |
```

Comment:
Maybe add something about it applies to Pod template file too, here and in values.yaml

```suggestion
| `workers.extraContainers`                             | Add additional containers to worker pod(s) including `pod_template_file`                                                                 | `[]`                                              |
```